Doc/data/refcounts.dat

```diff
@@ -724,6 +724,10 @@ PyErr_SetImportErrorSubclass:PyObject*:path:+1:
 
 PyErr_SetInterrupt:void:::
 
+_PyErr_SetLocaleString:int:::
+_PyErr_SetLocaleString:PyObject*:type:+1:
+_PyErr_SetLocaleString:const char*:message::
```

Comment:
I'm not sure that it's useful to document the refcount behavior of internal functions. Other functions prefixed by _Py are part of the public C API and/or documented.

Reply:
I don't think it hurts (or we can have a separate file for that). My motivation is to move towards a maintained `refcounts.dat` for every function, even those that are not part of the Stable ABI, but if you think it's better to do it later, I can remove that entry.